data/sql/updates/pending_db_world/haggle.sql

```diff
@@ -0,0 +1,124 @@
+(14041, 23, -19.452112, -3.621434, -4.2970552, NULL, 0, 'Haggle - sifting through trash 2'),
+(14041, 24, -14.487922, 0.329824, -4.287024, NULL, 0, 'Haggle'),
+(14041, 25, -15.168751, 20.166489, -4.3063, NULL, 0, 'Haggle'),
+(14041, 26, -16.66379, 21.67101, -4.305281, NULL, 0, 'Haggle'),
+(14041, 27, -18.431108, 21.27326, -4.304296, NULL, 0, 'Haggle'),
+(14041, 28, -18.431108, 21.27326, -4.304296, NULL, 0, 'Haggle - sifting through trash 3'),
+(14041, 29, -15.260129, 26.659346, -4.299999, NULL, 0, 'Haggle'),
+(14041, 30, -16.375502, 26.893675, -4.299803, NULL, 0, 'Haggle'),
+(14041, 31, -16.375502, 26.893675, -4.299803, NULL, 0, 'Haggle - sifting through trash 4'),
+(14041, 32, -13.471532, -15.474365, -4.2973704, NULL, 0, 'Haggle'),
+(14041, 33, -13.471532, -15.474365, -4.2973704, NULL, 0, 'Haggle - into sleeping on bench 2'),
+(14041, 34, -19.333311, -17.326876, -4.2973704, NULL, 0, 'Haggle'),
+(14041, 35, -19.333311, -17.326876, -4.2973704, NULL, 0, 'Haggle - start wander 3'),
+(14041, 36, -15.293428, -22.230337, -4.299684, NULL, 0, 'Haggle'),
+(14041, 37, -11.042005, -22.395052, -4.29911, NULL, 0, 'Haggle'),
+(14041, 38, -6.875696, -25.810347, -4.29732, NULL, 0, 'Haggle'),
+(14041, 39, -6.518164, -29.358128, -4.2977223, NULL, 0, 'Haggle'),
+(14041, 40, -4.615194, -32.011036, -4.2979712, NULL, 0, 'Haggle'),
+(14041, 41, 10.915832, -32.93334, -4.2973523, NULL, 0, 'Haggle'),
+(14041, 42, 14.343336, -28.164183, -4.2973585, NULL, 0, 'Haggle'),
+(14041, 43, 14.472878, -22.567085, -4.289472, NULL, 0, 'Haggle'),
+(14041, 44, 21.931023, -18.69022, -4.2973504, NULL, 0, 'Haggle'),
+(14041, 45, 22.149866, -17.656652, -4.2973504, NULL, 0, 'Haggle'),
+(14041, 46, 22.149866, -17.656652, -4.2973504, NULL, 0, 'Haggle - sifting through trash 5'),
+(14041, 47, 30.895535, -16.94747, -4.297347, NULL, 0, 'Haggle'),
+(14041, 48, 38.974773, -11.690686, -4.297347, NULL, 0, 'Haggle'),
+(14041, 49, 38.974773, -11.690686, -4.297347, NULL, 0, 'Haggle - sifting through trash 6'),
+(14041, 50, 37.514492, -1.188888, -4.2973495, NULL, 0, 'Haggle'),
+(14041, 51, 38.968258, 0.839936, -4.29735, NULL, 0, 'Haggle'),
+(14041, 52, 38.968258, 0.839936, -4.29735, NULL, 0, 'Haggle - sifting through trash 7'),
+(14041, 53, 27.43926, 5.252612, -4.297351, NULL, 0, 'Haggle'),
+(14041, 54, 21.324173, 13.69732, -4.2973514, NULL, 0, 'Haggle'),
+(14041, 55, 21.230026, 14.574246, -4.297351, NULL, 0, 'Haggle'),
+(14041, 56, 21.230026, 14.574246, -4.297351, NULL, 0, 'Haggle - sifting through trash 8'),
+(14041, 57, 27.30816, 14.715199, -4.2973504, NULL, 0, 'Haggle'),
+(14041, 58, 32.701, 21.962217, -4.2973495, NULL, 0, 'Haggle - talking'),
+(14041, 59, 38.99483, 24.375648, -4.2973485, NULL, 0, 'Haggle'),
+(14041, 60, 38.99483, 24.375648, -4.2973485, NULL, 0, 'Haggle - sifting through trash 9');
+
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 14041;
+
+DELETE FROM `smart_scripts` WHERE `entryorguid` IN (14041, 1404100, 1404101, 1404102, 1404103, 1404104);
```

Comment:
This is a big no-no
You always need to specify source_type when deleting smart_scripts, so divide into 2 DELETEs

Reply:
Ye I forgor. This is why this is a WIP